components/feature/intent/src/main/java/mozilla/components/feature/intent/ext/IntentExtensions.kt

```diff
@@ -8,6 +8,7 @@ import android.content.Intent
 import mozilla.components.support.utils.SafeIntent
 
 const val EXTRA_SESSION_ID = "activeSessionId"
+const val EXTRA_URL_OVERRIDE = "urlOverride"
```

Comment:
I'd like to move toward namespaced extras, so I'd prefer to use something like `mozilla.components.support.utils.extra.URL_OVERRIDE`. This matches the behaviour of the built-in Android actions & extras.